### ISSUE TITLE
Page attachment tap target is below minimum recommended size

### ISSUE THREAD

**Issue by jackbsteinberg** [ampproject/amphtml#25871]
The tap target for the arrow icon on the page attachment is 32x32px smaller than the minimum recommended size of 48x48px.

The tap target size should be increased to reflect this, and the gap between the icon and the text should be converted to padding to connect the tap targets of the two elements.

/cc @ampproject/wg-stories @newmuis @gmajoulet

**Comment by gmajoulet** [comment]
Adding some padding to the arrow icon sounds good.
But there shouldn't be any gap between the icon and the text, on what browser do you see this happening?

**Comment by jackbsteinberg** [comment]
The arrow icon and the text both open the page attachment when clicked, but there is a 12px margin in between the two that, when tapped or clicked, will just be treated as a normal click and advance the page (or select a quiz option in desktop mode if there's a quiz behind it).

I figured it might be good to change that gap to padding so that gap can be tapped and trigger the page attachment

**Comment by gmajoulet** [comment]
![image](https://user-images.githubusercontent.com/1492044/70174379-c5f67d80-16a2-11ea-9a82-d0faa13cc6c8.png)

I can click on the orange box and still open the attachment on mobile, but not on desktop :(
Good catch, thanks
